Feature/api update (#55)
hermeznetwork/docs: docs/users/mainnet.md

> # Hermez

Reviewer: ```suggestion
# Mainnet
```

Hermez is now live on Ethereum. 

## Tools
- [Web wallet](https://wallet.hermez.io/login) to interact with Hermez ZK-Rollup user accounts
- Hermez L2 [batch explorer](https://explorer.hermez.io/)
- Hermez [SDK](https://github.com/hermeznetwork/hermezjs)

> - Hermez [API](https://api.hermez.io/v1)

Reviewer: ```suggestion
- Hermez [API](https://api.hermez.io/v1/state)
```

## Useful Links

> - To interact with the Testnet, you'll need a [Metamask](https://metamask.io/) wallet.

Reviewer: ```suggestion
- To interact with hermez, you'll need a [Metamask](https://metamask.io/) wallet.
```

## Contract Addresses
Hermez uses 4 main smart contracts:
- HermezAuctionProtocol: [0x15468b45ed46c8383f5c0b1b6cf2ecf403c2aec2](https://etherscan.io/address/0x15468b45ed46c8383f5c0b1b6cf2ecf403c2aec2)
- HermezAddress: [0xa68d85df56e733a06443306a095646317b5fa633](https://etherscan.io/address/0xa68d85df56e733a06443306a095646317b5fa633)

> - HermezWithdrawalDelayeAddress: [0x392361427ef5e17b69cfdd1294f31ab555c86124](https://etherscan.io/address/0x392361427ef5e17b69cfdd1294f31ab555c86124)

Reviewer: ```suggestion
- HermezWithdrawalDelayerAddress: [0x392361427ef5e17b69cfdd1294f31ab555c86124](https://etherscan.io/address/0x392361427ef5e17b69cfdd1294f31ab555c86124)
```
